ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/util/RowUtil.java

```diff
@@ -32,4 +35,14 @@ private RowUtil() {
     }
     return valueToIndex;
   }
+
+  public static List<ColumnType> columnTypesFromStrings(final List<String> columnTypes) {
+    return columnTypes.stream().map(RowUtil::columnTypeFromString).collect(Collectors.toList());
+  }
+
+  private static ColumnType columnTypeFromString(final String columnType) {
```

Comment:
Perhaps push this into the ColumnType class?

static ColumnType of(String typeString) {
}

Reply:
You mean the `ColumnType` interface or the implementation class `ColumnTypeImpl`? The sort of string manipulation in this method feels odd in the interface. If we put it in the implementation, would it make more sense to instead update the constructor to perform the trimming, rather than having a static method for it? (Either's fine with me, just want to clarify what you had in mind.)